server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
What do we use this for? What happens when path has reserved regex characters in it?

I feel like it'd be simpler and less bug prone if we had an explicit route list.